unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.cs

```diff
@@ -155,12 +157,49 @@ public float GetDuration()
 
         public VideoState GetState()
         {
-            return (VideoState)plugin.GetState(videoPlayerId);
+            return plugin.GetState(videoPlayerId);
         }
 
         public void Dispose()
         {
             plugin.Remove(videoPlayerId);
         }
+
+
+        private void UpdateTextureConservingAspectRatio(Texture2D textureToCenter, bool fillBackground)
```

Comment:
Possible optimization of texture size: we can make the system force the texture passed to be understood as a square texture, this way, the texture does not need to be square and a good portion of the background can be removed.
The code would add a new parameter `bool useAsSquaredTexture`
When calculating the aspect ratio, if `useAsSquaredTexture` is `true`, only width would be used.